libraries/choregraphie.rb

```diff
@@ -44,10 +44,11 @@ def initialize(name, &block)
       # read all available primitives and make them available with a method
       # using their name. It allows to call `check_file '/tmp/titi'` to
       # instantiate the CheckFile primitive
+      delegated_args = RUBY_VERSION < '3' ? '*args, &block' : '*args, **kwargs, &block'
```

Comment:
Here and below please use `RUBY_VERSION.to_i < 3`; your code works ... until we get to ruby `10`

Reply:
Sure, will do! I hope we will able to drop ruby 2.7 support and remove this fix before Ruby v10, but yes :)